corehq/apps/export/static/export/js/export_list.js

```diff
@@ -185,6 +187,22 @@ hqDefine("export/js/export_list", [
             });
         };
 
+        var tooltipText = "";
+        if (self.isOData() || self.isFeed()) {
+            tooltipText = "All of the selected feeds will be deleted.";
```

Comment:
can you add `gettext` on all the user-facing text here? Thank you!

Reply:
added!